clippy_lints/src/mutex_atomic.rs

```diff
@@ -108,7 +120,7 @@ fn get_atomic_name(ty: Ty<'_>) -> Option<&'static str> {
                 UintTy::U32 => Some("AtomicU32"),
                 UintTy::U64 => Some("AtomicU64"),
                 UintTy::Usize => Some("AtomicUsize"),
-                // There's no `AtomicU128`.
+                // `AtomicU128` is unstable and only available on a few platforms: https://github.com/rust-lang/rust/issues/99069
```

Comment:
Maybe this lint could be configurable:
- You may want to specify a maximum size of integer that would trigger the lint (for example 32 bits) if you want your code to be portable across various architectures you use, with a reasonable default (32 or 64).
- You may want to check if the projected target type exist on the current target at least, by looking up the corresponding diagnostic item. Note that `AtomicI128` and `AtomicU128` are also diagnostic items, but you may also want to check for the "integer_atomics" feature before you are going to suggest them.